tests/users/test_api_update_user.py

```diff
@@ -79,6 +79,35 @@ def test_update_username_not_taken(self):
         self.assertDictEqual(expected_response, json.loads(actual_response.data))
         self.assertEqual(user1_new_username, self.first_user.username)
 
+    def test_update_username_same(self):
+
+        self.first_user = UserModel(
+            name=user1["name"],
+            email=user1["email"],
+            username=user1["username"],
+            password=user1["password"],
+            terms_and_conditions_checked=user1["terms_and_conditions_checked"],
+        )
```

Comment:
Use dictionary unpacking here
```suggestion
        self.first_user = UserModel(**user1)
```